helm/kubernetes-prometheus-chart/templates/prometheus-operator/crd-alertmanager.yaml

```diff
@@ -0,0 +1,2401 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
This CRD is really big. For my interest where does it come from?

Should the README mention which upstream repo this is based on?

Reply:
yeah let's add it. Yeah it comes from upstream